third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/DotOpToLLVM/MMAv5.cpp

```diff
@@ -438,17 +438,32 @@ void convertDotImpl(const LLVMTypeConverter &typeConverter,
   SmallVector<int64_t> shapeB = op.shapeB;
   SmallVector<unsigned> aOperandShape = {mmaSizeM, mmaSizeK};
 
+  auto getAllocShape = [&](MemDescType tensorTy, int kDim) {
+    // allocationShape uses the shape, not the `allocShape`?
+    auto fullAllocShape = triton::gpu::getAllocationShapePerCTA(
```

Comment:
I think each CTA takes a sub tensor of the allocated one?

Why the variable name is *full*AllocShape

Reply:
I didn't quite understand why `getAllocationShapePerCTA(Type)` uses `getShape` and not `getAllocShape` internally.